File: Front/src/app/models/banks/tran-data.ts
import { TranBase } from './tran-base';

export class TranData extends TranBase {
	public Code:string;
	public BankName:string;
	public TypeName:string;
	public CreatedDate:Date;
	public UpdatedDate:Date;
	public CurrentBalance:number;
	public CreatorUserID:string;
	public CreatorUserName:string;
	public ModifierUserID:string;
	public ModifierUserName:string;
}

File: Back/BancaNet.Backend.Framework/Models/Banks/TranData.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace BancaNet.Backend.Framework.Models.Banks
{
	public sealed class TranData : TranBase
	{
		public string Code { get; set; }
		public string BankName { get; set; }
		public string TypeName { get; set; }
		public DateTime? CreatedDate { get; set; }
		public DateTime? UpdatedDate { get; set; }
		public decimal? CurrentBalance { get; set; }
		public string CreatorUserID { get; set; }
		public string CreatorUserName { get; set; }
		public string ModifierUserID { get; set; }
		public string ModifierUserName { get; set; }
	}
}
